### DISCUSSION TITLE
Hi AI Ethics Specialists

### DISCUSSION THREAD

**Discussion by amritchhetrib78** [OpenMined/courses#70] [Introduce Yourself]
Hello ! This is Amrit Chhetri from Siliguri, India. Glad to be part of this great AI Ethics Community. Please connect me at https://in.linkedin.com/in/amritchhetrib

**Comment by iamtrask** [comment]
Hi Amrit! Welcome :)

**Reply by amritchhetrib78** [reply]
Thank you. Glad to connect and meet you here.
